Makefile.osx

```diff
@@ -90,14 +88,19 @@ installdebug:
 package:
 	build/package_mac;
 
-ci: 
-    mkdir build.release;     \
-    cd build.release;            \
-    cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DCMAKE_BUILD_TYPE=RELEASE \
-          -DCMAKE_BUILD_NUMBER="${BUILD_NUMBER}" -DBUILD_AUTOUPDATE=ON -DTELEMETRY_TRACK_ID="${TELEMETRY_TRACK_ID}" -DBUILD_UI_MU4="${BUILD_UI_MU4}" \
-          .. -G Xcode;                                                  \
-    xcodebuild -project ${XCODEPROJ} -target lrelease;                  \
-    $(FORMATTER_START) xcodebuild -project ${XCODEPROJ} -configuration Release -target install $(FORMATTER_END);
+ci:
+	@echo "Make ci"; \
+	mkdir build.release; \
+	cd build.release; \
+	cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+	-DCMAKE_BUILD_TYPE=RELEASE \
+	-DCMAKE_BUILD_NUMBER="${BUILD_NUMBER}" \
+	-DMUSESCORE_BUILD_CONFIG="${MUSESCORE_BUILD_CONFIG}" \
+	-DMUSESCORE_REVISION="${MUSESCORE_REVISION}" \
+	-DTELEMETRY_TRACK_ID="${TELEMETRY_TRACK_ID}" \
+	.. -G Xcode; \
```

Comment:
I'd rather indent all those options to `cmake`
It is done that way further up too

Reply:
fixed